src/elements/content-sidebar/__tests__/AddTaskButton.test.js

```diff
@@ -2,6 +2,21 @@ import React from 'react';
 import { mount } from 'enzyme';
 import { AddTaskButtonComponent as AddTaskButton } from '../AddTaskButton';
 
+jest.mock('../../../components/date-picker/DatePicker', () => props => {
```

Comment:
Will every test that renders a datepicker need a mock like this?

Reply:
I believe this was only necessary because `mount` was used for rendering in the test. So for consumers of date picker that use `mount` in their tests, yes we'll likely need to add a mock like this